pkg/cmd/root/root.go

```diff
@@ -93,6 +94,7 @@ func NewCmdRoot(f *cmdutil.Factory, version, buildDate string) *cobra.Command {
 	repoResolvingCmdFactory := *f
 	repoResolvingCmdFactory.BaseRepo = resolvedBaseRepo(f)
 
+	cmd.AddCommand(browseCmd.NewCmdBrowse(f)) // adds to the commands Commands()
```

Comment:
This will need to make use of a `repoResolvingCmdFactory` like the commands below it since `browse` makes use of `BaseRepo` (and that being confusing is on us, sorry. I forget to use `repoResolvingCmdFactory` all the time myself).

Reply:
the issue is fix, thanks for point it out for us.